websocketpp/extensions/permessage_deflate/enabled.hpp

```diff
@@ -213,7 +226,7 @@ class enabled {
       , m_server_max_window_bits_mode(mode::accept)
       , m_client_max_window_bits_mode(mode::accept)
       , m_initialized(false)
-      , m_compress_buffer_size(16384)
+      , m_compress_buffer_size(8192)
```

Comment:
Why reduce the buffer size?

Reply:
https://github.com/zaphoyd/websocketpp/commit/6330286ad5d56b918d8e32d32cb5860ee4619359